app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+def calculate_team_total_rating(team: list) -> int:
```

Comment:
You can provide more detailed type annotation
```suggestion
def calculate_team_total_rating(team: list[Player]) -> int:
```